crates/libp2p-networking/src/network/node.rs

```diff
@@ -299,13 +300,25 @@ impl NetworkNode {
                 ..Default::default()
             };
 
+            // Limit the number of simultaneous connections
+            let connection_limits = libp2p::connection_limits::Behaviour::new(
+                ConnectionLimits::default()
+                    .with_max_established(Some(100))
+                    .with_max_established_incoming(Some(70)) // To prevent eclipse attacks
```

Comment:
This might be trickier than we expect.  We might actually expect an incoming connection from every participant in the case of voting when the CDN is down.  We'll have to check to make sure that when we block some connection they can eventually get through.

Reply:
Yeah, even with the CDN being up this would be a problem